src/adafruit_circuitplayground/__init__.py

```diff
@@ -0,0 +1 @@
+from .express import cpx as cp
```

Comment:
Add a comment [pointing to their PR](https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/pull/79)?

Reply:
Added!